app/src/main/kotlin/com/wire/android/ui/home/newconversation/newgroup/NewGroupScreen.kt

```diff
@@ -121,7 +121,7 @@ fun NewGroupScreenContent(
                     }
 
                 }
-                if (mlsEnabled || BuildConfig.PRIVATE_BUILD) {
+                if (mlsEnabled || (BuildConfig.PRIVATE_BUILD && BuildConfig.MLS_SUPPORT_ENABLED)) {
```

Comment:
I think we can remove the private build here, and  only rely on the MLS_SUPPORT_ENABLED field.

Reply:
No we can't without changing the behaviour. 

We are controlling two separate things:
1. Is MLS support enabled
2. Is your user allowed to create MLS conversations 

The PRIVATE_BUILD is in this case by passing the team admin config which white lists users for creating MLS conversations.